airbyte-integrations/connectors/source-fleetio/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.9.11-alpine3.15 as base
```

Comment:
Since this is a new source, let's get it to use Airbyte-provided base image. That'd ensure that it's up to date, and will get docker image security updates as well, without any changes to the connector itself.

To get this done: 
- [ ] Remove the Dockerfile
- [ ] Set `connectorBuildOptions` in the `metadata.yaml`. [Example in `source-mailchimp`](https://github.com/airbytehq/airbyte/blob/d41cd621e77651af29e5d74cc0e0611514fb4c3a/airbyte-integrations/connectors/source-mailchimp/metadata.yaml#L8):

   ```
     connectorBuildOptions:
       baseImage: docker.io/airbyte/python-connector-base:1.1.0@sha256:bd98f6505c6764b1b5f99d3aedc23dfc9e9af631a62533f60eb32b1d3dbab20c
   ```

The current version is `1.2.0` though, double-check SHA on DockerHub.